src/components/Button/Button.module.css

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 .button {
   border-radius: var(--cpd-radius-pill-effect);
   cursor: pointer;
-  -webkit-appeareance: none;
```

Comment:
That was a typo not being caught by stylelint because of the shell expansion issue